vaadin-renderer-flow-parent/vaadin-renderer-flow/src/main/resources/META-INF/resources/frontend/flow-component-directive.js

```diff
@@ -11,22 +11,28 @@ class FlowComponentDirective extends Directive {
   }
 
   update(part, [appid, nodeid]) {
+    this.updateContent(part, appid, nodeid);
+    return noChange;
+  }
+
+  updateContent(part, appid, nodeid) {
     const { parentNode, startNode } = part;
 
     const newNode = this.getNewNode(appid, nodeid);
     const oldNode = this.getOldNode(part);
 
-    if (oldNode === newNode) {
-      return noChange;
+    if (!newNode) {
+      // If the node is not found, try again later.
+      setTimeout(() => this.updateContent(part, appid, nodeid));
```

Comment:
Looking at the code, it seems the retry logic can be prone to infinite loops. However, since it is just a replication of the previous implementation, I think it is ok for now.